ep0099/doc.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that caan be found in the LICENSE file.
+
+// Package ep0099
+//
+// Datasheet
```

Comment:
Can you tell which chip is being used. I almost see it at https://wiki.52pi.com/images/2/2e/Ddl-4.jpg but

Reply:
The closest I could get was at https://wiki.52pi.com/images/5/52/Ddl-11.jpg

Apparently it reads "32F030F4P6".
https://www.st.com/en/microcontrollers-microprocessors/stm32f030f4.html#cad-resources

Reply:
Thanks.